fidesctl/src/fidesctl/cli/commands/util_comands.py

```diff
@@ -0,0 +1,133 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
+    """
+    Initialize a Fidesctl instance.
+    """
+
+    # Constants
+    dir_name = ".fides"
+    config_path = f"{dir_name}/fidesctl.toml"
+    config = ctx.obj["CONFIG"]
+
+    # List the values we want to exclude from the user-facing config
+    # These are values that are generated at config load time
+    excluded_values = {
```

Comment:
Would love feedback on what values to exclude here. Should the test_database_url be excluded too?

Reply:
Why not accept a list of options for things to _include_ instead? My ideal usage would look like:

```
usage: init config_path [options...]

	Configure a new fidesctl instance by creating the necessary .fides directory
	and its structure. The .fides directory is placed within the directory specified
	by config_path. If a .fides directory already exists within config_path, no
	action is taken.

	A fidesctl.toml configuration file will be created in the .fides directory, but
	will be left empty by default. To automatically populate this file with the
	desired configuration options and their values, include any combination and
	amount of the available command line options documented below. For an example
	of this file's expected structure, and a list of available fidesctl.toml
	configuration options, see:
		https://ethyca.github.io/fides/installation/configuration/

options:
	-a, --api_option name value
	-c, --cli_option name value
	-u, --user_option name value
```

and I would expect to call the command like:

```sh
fidesctl init ~/repository_root \
 -a database_url "postgres:fidesctl@fidesctl-db:5432/fidesctl_test" \
 -a log_level "DEBUG"
 -c server_url "http://localhost:8080"
```

The result would be the following file structure (some additional empty directories include here for manifest files):

```
~/repository_root/
└── .fides/
    ├── datasets/
    ├── policies/
    ├── systems/
    └── fidesctl.toml
```

and the `fidesctl.toml` file would contain:

```toml
[api]
database_url = "postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
log_level = "DEBUG"

[cli]
server_url = "http://localhost:8080"
```